### PR TITLE
Add cls token to BEHRT embedder

### PR DESCRIPTION
**Pull request by frillecode**
# Notes for reviewers
Reviewers can skip X, but should pay attention to Y.

### CODE REVIEW

File: psycop/common/sequence_models/aggregators.py [hunk 1]
```diff
@@ -14,6 +14,22 @@ def forward(
         pass
 
 
+class CLSAggregationModule(AggregationModule):
```
Comment (frillecode) [inline psycop/common/sequence_models/aggregators.py:17]: rename

Comment (KennethEnevoldsen) [reply psycop/common/sequence_models/aggregators.py:17]: I think it is fine

File: psycop/common/sequence_models/tests/test_encoder_for_clf.py [hunk 2]
```diff
@@ -56,11 +56,11 @@ def encoder_module() -> nn.Module:
...
 
 @pytest.fixture()
-def aggregation_module() -> AveragePooler:
+def aggregation_module() -> CLSAggregationModule:
     """
-    A mean pooling module
+    Aggregation module that takes the hidden state of the first token (i.e. the CLS token)
     """
-    return AveragePooler()
+    return CLSAggregationModule()
```
Comment (KennethEnevoldsen) [inline psycop/common/sequence_models/tests/test_encoder_for_clf.py:63]: I would test both aggregation modules

Comment (KennethEnevoldsen) [reply psycop/common/sequence_models/tests/test_encoder_for_clf.py:63]: you can do this using pytest.mark.parametrize.

**Review by KennethEnevoldsen — APPROVED** [review]
Looks great. Only a minor comment on tests

File: psycop/common/sequence_models/tests/test_encoder_for_clf.py (hunk 2 above)
Comment (frillecode) [reply psycop/common/sequence_models/tests/test_encoder_for_clf.py:63]: very cool, didnt know that

**Pull request merged**